pkg/api/addons_test.go

```diff
@@ -7424,23 +7403,25 @@ func TestSetAddonsConfig(t *testing.T) {
 					Containers: []KubernetesContainerSpec{
 						{
 							Name:  common.CoreDNSAddonName,
-							Image: specConfig.KubernetesImageBase + K8sComponentsByVersionMap["1.15.4"][common.CoreDNSAddonName],
+							Image: specConfig.KubernetesImageBase + K8sComponentsByVersionMap["1.13.11"][common.CoreDNSAddonName],
 						},
 					},
 				},
 			},
 		},
 		{
-			name: "kube-dns enabled",
+			name: "upgrade w/ manual kube-dns enabled",
```

Comment:
This is the "user manually modifies his/her api model to explicitly continue using kube-dns, which is now an option since making kube-dns a user-configurable addon"